sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/AnyValue.scala

```diff
@@ -32,7 +32,8 @@ import org.apache.spark.sql.types._
 @ExpressionDescription(
   usage = """
     _FUNC_(expr[, isIgnoreNull]) - Returns some value of `expr` for a group of rows.
-      If `isIgnoreNull` is true, returns only non-null values.""",
+      If `isIgnoreNull` is true, returns only non-null values.
```

Comment:
can you point to the code where we ignore trailing `\n` for DESC FUNCTION?

Reply:
Take this comment as example, `getUsage` of `ExpressionInfo` outputs show below.
![image](https://user-images.githubusercontent.com/8486025/180951837-03a77079-a1ec-490b-9dd6-d73d033e6002.png)
no trailing `\n` here.

Reply:
are you saying the scala multiline syntax won't produce trailing `\n`?
```
"""
str
"""
```

Reply:
Yes.